### ISSUE TITLE
centos7: migrate db django error

### ISSUE THREAD

**Issue by juju4** [zentralopensource/zentral#78]
With latest changes from Mar 27, migrate db failed on centos7
```
       TASK [juju4.zentral : migrate db] **********************************************
       task path: /tmp/kitchen/juju4.zentral/tasks/db.yml:13
       fatal: [localhost]: FAILED! => {"changed": true, "cmd": ["python3", "manage.py", "migrate"], "delta": "0:00:00.283813", "end": "2019-03-28 03:45:17.018815", "msg": "non-zero return code", "rc": 1, "start": "2019-03-28 03:45:16.735002", "stderr": "Traceback (most recent call last):\n  File \"manage.py\", line 8, in <module>\n    from django.core.management import execute_from_command_line\n  File \"/usr/lib64/python3.4/site-packages/django/core/management/__init__.py\", line 11, in <module>\n    from django.conf import settings\n  File \"/usr/lib64/python3.4/site-packages/django/conf/__init__.py\", line 18, in <module>\n    from django.utils.functional import LazyObject, empty\n  File \"/usr/lib64/python3.4/site-packages/django/utils/functional.py\", line 12\n    return _curried_func(*args, *moreargs, **{**kwargs, **morekwargs})\n                                ^\nSyntaxError: invalid syntax", "stderr_lines": ["Traceback (most recent call last):", "  File \"manage.py\", line 8, in <module>", "    from django.core.management import execute_from_command_line", "  File \"/usr/lib64/python3.4/site-packages/django/core/management/__init__.py\", line 11, in <module>", "    from django.conf import settings", "  File \"/usr/lib64/python3.4/site-packages/django/conf/__init__.py\", line 18, in <module>", "    from django.utils.functional import LazyObject, empty", "  File \"/usr/lib64/python3.4/site-packages/django/utils/functional.py\", line 12", "    return _curried_func(*args, *moreargs, **{**kwargs, **morekwargs})", "                                ^", "SyntaxError: invalid syntax"], "stdout": "", "stdout_lines": []}
```
https://travis-ci.org/juju4/ansible-zentral/jobs/512250120#L5300

works fine on ubuntu xenial
https://travis-ci.org/juju4/ansible-zentral/jobs/512250121#L4401
and was working fine 10 days ago on both.

issue seems with django but requirements update to 2.1 is a month old and 2.1.7 was released on Feb 11 (https://github.com/django/django/releases). previous builds worked fine.
Ideas?

Thanks

**Comment by np5** [comment]
Django 2.1 only supports python > 3.5. The error is triggered because of the [new unpacking capabilities available in python 3.5](https://docs.python.org/3.5/whatsnew/3.5.html#whatsnew-pep-448).

**Comment by juju4** [comment]
Thanks @np5 ! switched to epel python36 and works back!
